docker-compose.yml

```diff
@@ -1,3 +1,318 @@
-# Stub file to get paths in all docker-compose files to work from project root.
-# Need this per: https://github.com/docker/compose/issues/3874
-version: '3.7'
+services:
+  startup:
+    extends:
+      file: common.yml
+      service: oada-uservice
+    image: oada/startup:${OADA_VERSION-build}
+    build:
+      args:
+        OADA_SERVICE: startup
+    container_name: startup
+    depends_on:
+      - zookeeper
+      - kafka
+      - arangodb
+    expose: # expose only internally, not on host
+      - '80'
+    environment:
+      # set RESETDATABASE to "yes" if you want to drop database on startup and recreate
+      - RESETDATABASE=${RESETDATABASE:-no}
+
+  auth:
+    extends:
+      file: common.yml
+      service: oada-uservice
+    image: oada/auth:${OADA_VERSION-build}
+    build:
+      args:
+        OADA_SERVICE: auth
+    container_name: auth
+    depends_on:
+      - startup
+    networks:
+      - http_net
+    expose: # expose only internally, not on host
+      - '80'
+    volumes:
+      - ./domains-available:/domains-available
+      - ./domains-enabled:/domains-enabled
+
+  # http-handler is in charge of maintaining connectiongs to clients and starting
+  # the first message for a request into Kafka
+  http-handler:
+    extends:
+      file: common.yml
+      service: oada-uservice
+    image: oada/http-handler:${OADA_VERSION-build}
+    build:
+      args:
+        OADA_SERVICE: http-handler
+    container_name: http-handler
+    depends_on:
+      - startup
+    networks:
+      - http_net
+    expose: # expose only internally, not on host
+      - '80' # at one time this mapped 34135 from host to local 80 for some reason
+    environment:
+      - IGNORE_SCOPE=${IGNORE_SCOPE:-}
+
+  sync-handler:
+    extends:
+      file: common.yml
+      service: oada-uservice
+    image: oada/sync-handler:${OADA_VERSION-build}
+    build:
+      args:
+        OADA_SERVICE: sync-handler
+    container_name: sync-handler
+    depends_on:
+      - startup
+      - proxy
+    networks:
+      - http_net
+    environment:
+      - IGNORE_SCOPE=${IGNORE_SCOPE:-}
+
+  write-handler:
+    extends:
+      file: common.yml
+      service: oada-uservice
+    image: oada/write-handler:${OADA_VERSION-build}
+    build:
+      args:
+        OADA_SERVICE: write-handler
+    container_name: write-handler
+    depends_on:
+      - startup
+
+  users:
+    extends:
+      file: common.yml
+      service: oada-uservice
+    image: oada/users:${OADA_VERSION-build}
+    build:
+      args:
+        OADA_SERVICE: users
+    container_name: users
+    depends_on:
+      - startup
+
+  token-lookup:
+    extends:
```

Comment:
I thought I already removed this service from docker-compose.  It is dead, I need to go through and clean these up after accepting PR.